### ISSUE TITLE
Issue with clicking on elements

### ISSUE THREAD

**Issue by bcccgreen** [giotiskl/Filterizr#58]
I had an issue with our grid, in that the hidden elements were sitting above the displayed elements, intercepting their clicks.

I ended up writing this function which sets the z-index for the cards we want above the others.  Hope it helps someone else, or someone can point out what I was doing wrong which caused the issue.

Call this on the click handler for your category headings.  Our filtr-items also have the class "card".

```
  /**
   * Issues with filterizr not hiding the cards but just
   * making them opacity=0 meant clicking would often hit the wrong
   * element.  Modifying the z-index seems to fix this.
   */
  function resetZIndex() {
    var active = $('ul.filter button.active').attr('data-filter');
    $('div.card').each(function () {
      var data_category = $(this).attr('data-category');
      var data_filters = data_category.split(', '); // We have a space here, you may not
      var z_index = 0;
      for (var i = 0, l = data_filters.length; i < l; i++) {
        if (data_filters[i] === active) {
          z_index = 1;
        }
      }
      $(this).css('zIndex', z_index);
    });
  }
```

**Comment by gpra8449** [comment]
Did you find the issue was on all browsers?

**Comment by giotiskl** [comment]
This issue is now natively addressed in [1.3.0](https://github.com/giotiskl/Filterizr/releases/latest).
